Documentation/troubleshooting.md

```diff
@@ -51,7 +51,7 @@ A common problem related to `ServiceMonitor` identification by Prometheus is rel
 
 <!-- do not change this link without verifying that the image will display correctly on https://prometheus-operator.dev -->
```

Comment:
this comment says it all :)

Reply:
Apologies, didn't quite understand it until digging into the site a bit further! I have updated the comments link to make it abundantly clear.
Any way to independently fix the github doc link?

Reply:
> Any way to independently fix the github doc link?

If you mean fixing the link so it works properly in GitHub, the answer is no (at least I have no solution).

Reply:
fair enough!